### PR TITLE
CLI: Add --root and --rootAlias options

### PR DESCRIPTION
**Pull request by aaronmccall**
For projects that use an alias for project root in their imports

ref: #7

### CODE REVIEW

File: cli/cli.js [hunk 1]
```diff
@@ -130,8 +135,17 @@ function traverseFiles(file) {
...
     case 'ImportDeclaration':
-      const isRelativePath = node.source.value.startsWith('.');
-      const importPath = path.resolve(isRelativePath ? basePath : NODE_PATH, node.source.value);
+      let filepath = node.source.value
+      let dirpath = NODE_PATH
+      if (flags.rootAlias && filepath.indexOf(flags.rootAlias) === 0) {
+        filepath = filepath.replace(/^~\/?/, '')
+        dirpath = flags.root
+      }
+      if (filepath.startsWith('.')) {
```
Comment (andreruffert) [inline cli/cli.js]: What do you think about to keep the `isRelativePath` variable for a better understanding?
https://github.com/andreruffert/i18n-literally/pull/8/files#diff-d72e128c30ac15619754c48d8662b401L133

**Comment by aaronmccall** [comment]
@andreruffert,

Thanks for the feedback! I've addressed all of your requested changes.
Have a great evening!

**Comment by aaronmccall** [comment]
@andreruffert, what's the status on this? I believe I've addressed every requested change.

**Comment by andreruffert** [comment]
Published in [v1.7.0](https://github.com/andreruffert/i18n-literally/releases/tag/v1.7.0).

**Pull request merged**